### PR TITLE
Restrict slack events

### PR DESCRIPTION
**Pull request by joshblum**

### CODE REVIEW

File: orchestra/communication/slack.py [hunk 1]
```diff
@@ -32,14 +32,14 @@ def post_message(self, slack_user_id, message, parse='none'):
 
 
 def get_slack_user_id(slack_username):
-    slack = SlackService()
+    slack = OrchestraSlackService()
```
Comment (joshblum) [inline orchestra/communication/slack.py:36]: add run_if here as well

File: example_project/example_project/orchestra_settings.py [hunk 2]
```diff
@@ -185,6 +183,10 @@ def setup_orchestra(settings_module_name):
     # Feature flags for toggling optional slack integration
     settings.SLACK_INTERNAL = False
```
Comment (joshblum) [inline example_project/example_project/orchestra_settings.py]: refactor: `ORCHESTRA_SLACK_INTERNAL_ENABLED`

File: example_project/example_project/orchestra_settings.py [hunk 3]
```diff
@@ -185,6 +183,10 @@ def setup_orchestra(settings_module_name):
...
     settings.SLACK_EXPERTS = False
```
Comment (joshblum) [inline example_project/example_project/orchestra_settings.py]: refactor: `ORCHESTRA_SLACK_EXPERTS_ENABLED`

File: example_project/example_project/orchestra_settings.py [hunk 4]
```diff
@@ -185,6 +183,10 @@ def setup_orchestra(settings_module_name):
...
+    settings.ORCHESTRA_SEND_STAFFING_MESSAGES = False
```
Comment (joshblum) [inline example_project/example_project/orchestra_settings.py]: refactor: `ORCHESTRA_SLACK_ACTIONS_ENABLED`

File: orchestra/communication/slack.py (hunk 1 above)
Comment (joshblum) [inline orchestra/communication/slack.py:36]: done

File: example_project/example_project/orchestra_settings.py (hunk 2 above)
Comment (joshblum) [inline example_project/example_project/orchestra_settings.py]: done

File: example_project/example_project/orchestra_settings.py (hunk 3 above)
Comment (joshblum) [inline example_project/example_project/orchestra_settings.py]: done

File: example_project/example_project/orchestra_settings.py (hunk 4 above)
Comment (joshblum) [inline example_project/example_project/orchestra_settings.py]: done

**Comment by joshblum** [comment]
Will need to clean up other settings in a future PR: https://github.com/unlimitedlabs/orchestra/issues/269

**Comment by marcua** [comment]
lgtm

**Pull request merged**